main/src/main/scala/sbt/Defaults.scala

```diff
@@ -2603,6 +2603,7 @@ object Defaults extends BuildCommon {
   lazy val testSettings: Seq[Setting[_]] = configSettings ++ testTasks
 
   @nowarn
+  @deprecated("Create a separate subproject instead of using IntegrationTest and in addition avoid using itSettings", "1.9.0")
```

Comment:
Apparently this file needs formatting.

Reply:
Will do it tonight when I get back to laptop

Reply:
Pushed and fixed.